app/components/Upload.js

```diff
@@ -444,6 +529,41 @@ export default class Upload extends Component {
     );
   }
 
+  renderMedtronic600SerialNumberInput() {
+    const { upload } = this.props;
+    if (_.get(upload, 'source.driverId', null) !== 'Medtronic600') {
+      return null;
+    }
+
+    return (
+      <div>
+        <div className={styles.textInputWrapper}>
+          <div className={styles.rememberWrap}>
+            <input
+              type="checkbox"
+              id="medtronic600Linked"
+              onChange={this.onMedtronic600LinkedChange}
+              checked={this.state.medtronic600Linked} />
+            <label htmlFor="medtronic600Linked">
+              {this.props.text.MEDTRONIC_600_IS_LINKED}
+            </label>
+          </div>
+          <div style={!this.state.medtronic600Linked ?
```

Comment:
I think the preferred style through the codebase is to use CSS classname styles via `styles` and assemble the classnames using `classnames` (generally imported as `cx`) instead of inlining the ternary here.